src/js/utils/create-logger.js

```diff
@@ -33,6 +33,11 @@ const LogByTypeFactory = (name, log) => (type, level, args) => {
   // Add a clone of the args at this point to history.
   if (history) {
     history.push([].concat(args));
+
+    // remove old history
+    while (history.length > 1000) {
```

Comment:
would something like `history = history.slice(-1000)` work? Theoretically, should be faster as well, since we won't be shifting the entire list n times.

Reply:
makes sense to me